Braintree/API/Networking/BTURLUtils.m

```diff
@@ -48,12 +48,7 @@ + (NSString *)queryStringWithDictionary:(NSDictionary *)dict {
 }
 
 + (NSString *)stringByURLEncodingAllCharactersInString:(NSString *)aString {
-    NSString *encodedString = (__bridge_transfer NSString * ) CFURLCreateStringByAddingPercentEscapes(NULL,
-                                                                                                      (__bridge CFStringRef)aString,
-                                                                                                      NULL,
-                                                                                                      (CFStringRef)@"&()<>@,;:\\\"/[]?=+$|^~`{}",
-                                                                                                      kCFStringEncodingUTF8);
-    return encodedString;
+    return [aString stringByAddingPercentEncodingWithAllowedCharacters:[NSCharacterSet URLQueryAllowedCharacterSet]];
```

Comment:
Since this method is used to encode keys and values, we need to also be sure to also encode the reserved characters (`;/?:@&=+$,`).